app/policies/application_policy.rb

```diff
@@ -6,6 +6,84 @@ def initialize(user, record)
     @record = record
   end
 
+  # Checks if the primary person of the given family is an individual market primary family member.
+  # A primary person is considered an individual market primary family member if they have either a consumer role or a resident role,
+  # and the user is the same as the user of the primary person.
+  #
+  # @param family [Family] The family to check.
+  # @return [Boolean] Returns true if the primary person of the family is an individual market primary family member, false otherwise.
+  def individual_market_primary_family_member?(family)
+    (family.primary_person.consumer_role.present? || family.primary_person.resident_role.present?) && user == family.primary_person.user
+  end
+
+  # Checks if the user is associated with an active broker for the given family.
+  # A user is considered associated with an active broker if the broker is not blank, is active,
+  # and matches the broker associated with the active broker agency account of the family.
+  #
+  # @param family [Family] The family to check.
+  # @return [Boolean] Returns true if the user is associated with an active broker for the family, false otherwise.
+  def active_associated_family_broker?(family)
+    broker = user.person.broker_role
+    return false if broker.blank? || !broker.active?
+
+    broker_agency_account = family.active_broker_agency_account
+    return false if broker_agency_account.blank?
+
+    broker_agency_account.benefit_sponsors_broker_agency_profile_id == broker.benefit_sponsors_broker_agency_profile_id &&
+      broker_agency_account.writing_agent_id == broker.id
+  end
+
+  # Checks if the primary person of the given family has their identity verified.
+  # The method assumes that the consumer_role is already defined and belongs to the primary person of the family.
+  #
+  # @param family [Family] The family to check.
+  # @return [Boolean] Returns true if the primary person of the family has their identity verified, false otherwise.
+  def ridp_verified_primary_person?(family)
```

Comment:
- We are initializing FamilyPolicy with family record. 
- Since record itself is family in application policy, do we have to pass family to this method?